app/components/dashboard/moab_storage_roots_component.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Dashboard
+  # methods for dashboard pertaining to MoabStorageRootsComponent
+  class MoabStorageRootsComponent < ViewComponent::Base
+    attr_reader :dashboard_catalog_service
+
+    delegate :status_labels,
```

Comment:
I wonder if it wouldn't be easier/cleaner to turn each of these services into a mixin.